### PR TITLE
Update CSV file and CR samples with current API and images.

### PR DESCRIPTION
**Pull request by shivamerla**

### CODE REVIEW

File: .github/workflows/image.yml [hunk 1]
```diff
@@ -128,5 +128,5 @@ jobs:
           BUNDLE_IMAGE: ghcr.io/nvidia/k8s-nim-operator:${{ env.BRANCH_NAME }}
           VERSION: '' # Until we cut a release, Version will be null
         run: |
-          sed -i "s|nvcr.io/nvidia/cloud-native/k8s-nim-operator:v0.1.0|${BUNDLE_IMAGE}|g" bundle/manifests/k8s-nim-operator.clusterserviceversion.yaml
+          sed -i "s|ghcr.io/nvidia/k8s-nim-operator:main|${BUNDLE_IMAGE}|g" bundle/manifests/k8s-nim-operator.clusterserviceversion.yaml
```
Comment (ArangoGutierrez) [inline .github/workflows/image.yml:131]: THis change is not related to the commit message or PR title

File: config/samples/apps_v1alpha1_nimservice.yaml [hunk 2]
```diff
@@ -3,12 +3,12 @@ kind: NIMService
...
   labels:
     app.kubernetes.io/name: k8s-nim-operator
-    app.kubernetes.io/managed-by: kustomize
+    app.kubernetes.io/managed-by: k8s-nim-operator
   name: meta-llama3-8b-instruct
 spec:
   image:
     repository: nvcr.io/nim/meta/llama3-8b-instruct
-    tag: 1.0.0
+    tag: 1.0.3
```
Comment (ArangoGutierrez) [inline config/samples/apps_v1alpha1_nimservice.yaml:11]: THis change is not related to the commit message or PR title

File: config/samples/apps_v1alpha1_nimcache.yaml [hunk 3]
```diff
@@ -3,12 +3,12 @@ kind: NIMCache
...
   labels:
     app.kubernetes.io/name: k8s-nim-operator
-    app.kubernetes.io/managed-by: kustomize
+    app.kubernetes.io/managed-by: k8s-nim-operator
   name: nimcache-sample
 spec:
   source:
     ngc:
-      modelPuller: nvcr.io/nim/meta/llama3-8b-instruct:1.0.0
+      modelPuller: nvcr.io/nim/meta/llama3-8b-instruct:1.0.3
```
Comment (ArangoGutierrez) [inline config/samples/apps_v1alpha1_nimcache.yaml:11]: THis change is not related to the commit message or PR title

**Review by ArangoGutierrez — CHANGES_REQUESTED** [review]

File: .github/workflows/image.yml (hunk 1 above)
Comment (shivamerla) [reply .github/workflows/image.yml:131]: it is related. This assumes that the image in the CSV file is always versioned `v0.1.0`, that is changing with the commit.

**Review by ArangoGutierrez — APPROVED** [review]

**Comment by ArangoGutierrez** [comment]
I am ok with the title rename
